launchdarkly/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "launchdarkly",
+  "manifest_version": "1.0.0",
+  "maintainer": "support@launchdarkly",
+  "display_name": "LaunchDarkly",
+  "short_description": "Track metrics from LaunchDarkly's relay proxy",
+  "metric_prefix": "launchdarkly.",
+  "metric_to_check": "launchdarkly.test",
```

Comment:
Is the metric to check actually `launchdarkly.test`? I notice its not in the metadata.csv file, is this some healthcheck that the integration sends, and is this sent regularly/consistently?